src/main/java/uk/gov/companieshouse/service/AppealService.java

```diff
@@ -87,7 +87,7 @@ private String createAppealInMongoDB(Appeal appeal, String userId) {
         } else {
             LOGGER.infoContext(penaltyReference,
                     "Update existing appeal record with reason: " + queryResult.isEmpty(), null);
-            Appeal updatedAppeal = this.appealMapper.map(queryResult.get(0));
+            var updatedAppeal = this.appealMapper.map(queryResult.get(0));
```

Comment:
Sonar requires us to change from Appeal to var

Reply:
I'm assuming this is because it's java11?

Reply:
It seems this Sonar issue is from Java 10 onwards

I would like to disagree with Sonar, if I were given a choice